src/storage/src/hummock/sstable/builder.rs

```diff
@@ -115,11 +115,12 @@ impl SSTableBuilder {
         // TODO: refine me
         let mut raw_value = BytesMut::default();
         let value_meta = value.encode(&mut raw_value) & ((1 << VNODE_BITS) - 1);
-        let table_id = get_table_id(full_key);
-        self.vnode_bitmaps
-            .entry(table_id)
-            .or_insert([0; VNODE_BITMAP_LEN])[(value_meta >> 3) as usize] |=
-            1 << (value_meta & 0b111);
+        if let Some(table_id) = get_table_id(full_key) {
+            self.vnode_bitmaps
+                .entry(table_id)
+                .or_insert([0; VNODE_BITMAP_LEN])[(value_meta >> 3) as usize] |=
+                1 << (value_meta & 0b111);
```

Comment:
Could you please add some comments for this?

Reply:
fixed